readers-frontend/dist/index.html

```diff
@@ -83,64 +81,180 @@
     </li>
   </ul>
 </nav>
+    <main class="pt-8 2xl:px-[13%] px-[0.5%]">
+		
+
+<!-- 1 article --><div class="flex w-full gap-2.5">
+    <div class="max-w-[50%] border-r-2 pr-2.5 border-gray-300">
+      
+
+
+<div class="h-full text-ellipsis overflow-hidden astro-Q7KXOHAS">
+  <div class="flex astro-Q7KXOHAS" style="--categoryColor: #000000;">
+      <a class="w-1/2 h-full  astro-Q7KXOHAS" href="/2022/09/04/featuredArticles[0].articles_slug">
```

Comment:
I don't think this is meant to be a string?
Shouldn't it be
```js
`/2022/09/04/${featuredArticles[0].articles_slug}`
```

There's a couple more occurrences of the same

Reply:
I mean I picked one from the dist folder, but that's coming from the source

Reply:
Yup You're right! I fixed it now